tftypes/value_json.go

```diff
@@ -422,7 +442,11 @@ func jsonUnmarshalTuple(buf []byte, elementTypes []Type, p *AttributePath) (Valu
 	}, vals), nil
 }
 
-func jsonUnmarshalObject(buf []byte, attrTypes map[string]Type, p *AttributePath) (Value, error) {
+// jsonUnmarshalObject attempts to decode JSON object structure to tftypes.Value object.
+// opts contains fields that can be used to modify the behaviour of JSON unmarshalling.
+// ignoreUndefinedAttributes is used to ignore any attributes which appear in the JSON but do not have a corresponding
+// entry in the schema. For example, raw state where an attribute has been removed from the schema.
```

Comment:
Nit: Can you move this up to the Go documentation for the options field? Thanks!

Reply:
Moved.